definitions/procedures/foreman_openscap/invalid_report_associations.rb

```diff
@@ -0,0 +1,15 @@
+module Procedures::ForemanOpenscap
+  class InvalidReportAssociations < ForemanMaintain::Procedure
+    metadata do
+      param :ids_to_remove, 'Ids of reports to remove'
```

Comment:
@xprazak2 ,

param ` :ids_to_remove` is required to run this procedure so you could mention `:required => true` to param declaration.